### PR TITLE
fix(reports): prune empty balance accounts

### PR DESCRIPTION
**Pull request by jniles**
Fixes a bug in the balance report to ensure pruning accounts takes place.  Also fixes the print orientation of the page.

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda can I get a review?

File: server/controllers/finance/reports/balance/index.js [hunk 1]
```diff
@@ -236,13 +231,14 @@ function computeBalanceTree(accounts, totals, context, shouldPrune) {
...
   tree.walk(Tree.common.computeNodeDepth);
 
-  // prune empty rows if needed
+  let balances;
+  // prune empty rows if needed (prune() does not work in-place)
   if (shouldPrune) {
-    tree.prune(isEmptyRow);
+    balances = tree.prune(isEmptyRow);
+  } else {
+    balances = tree.toArray();
```
Comment (mbayopanda) [inline server/controllers/finance/reports/balance/index.js:239]: 👍

**Review by mbayopanda — APPROVED** [review]
LGTM 👍

**Pull request merged**